integration-cli/docker_cli_daemon_test.go

```diff
@@ -1193,6 +1193,19 @@ func (s *DockerDaemonSuite) TestDaemonLoggingDriverNoneLogsError(c *check.C) {
 	c.Assert(out, checker.Contains, expected)
 }
 
+func (s *DockerDaemonSuite) TestDaemonLoggingDriverNoneWithBuild(c *check.C) {
```

Comment:
I was considering something like `// Test for Issue 19259: the daemon log config should be ignored when building images`.
But it is ok to use a descriptive name. 

Just IMO, `TestDaemonLoggingDriverShouldBeIgnoredForBuild` would be more descriptive? (too verbose? 😅 )